### PR TITLE
Moves docs to paradox

### LINKED ISSUE #20: Setup paradox

### PR DESCRIPTION
**Pull request by ignasi35**
Fixes #20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,99 +1,16 @@
-# play-scala-grpc-example
+# Play Scala gRPC Example
+
+This example is described in the [Play Scala gRPC Example site](https://developer.lightbend.com/guides/play-scala-grpc-example/).
```
Comment (ignasi35) [inline README.md:3]: moved almost all the content in `README` to `docs/src/main/paradox/index.md`.

Reviewed the remaining bits and added the trailing section`Sample License`

File: project/paradox.sbt [hunk 2]
```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.3.5")
```
Comment (ignasi35) [inline project/paradox.sbt:1]: Could bump to `0.4.2` but I'm keeping it in sync with Lagom-scala-gRPC-example code. We'll upgrade with `templatecontrol`

**Review by dwijnand — APPROVED** [review]
Thanks!

**Comment by ignasi35** [comment]
Dear GH-travis-check, the [job](https://travis-ci.org/playframework/play-scala-grpc-example/builds/478448931?utm_source=github_status&utm_medium=notification) finished successfully **3 DAYS AGO**!!

screen shot 2019-01-14 at 10 29 29

**Pull request merged**